### DISCUSSION TITLE
Use as local relay mailserver along with different docker container

### DISCUSSION THREAD

**Discussion by sakearzoo** [docker-mailserver/docker-mailserver#2693] [Q&A]
i have a plan to use it as a normal mail server as well as local mail relay server.
i have a domain named `example.com` and my container **anonaddy/simplelogin** will use this container as a relay server locally but i need to have few email id setup with docker-mailserver like `report@example.com/support@example.com` where i can directly use smtp/imap to send and receive mail. whats the proper way of setting it up. i don't want to have relay server exposed publicly . only my docker network could use and access the mail-server to use it as a relay server.

i am a bit confused . any suggestion would be helpful and example configuration will extremely helpful

**Comment by polarathene** [comment]
I shared a [basic offline example](https://github.com/docker-mailserver/docker-mailserver/issues/2601#issuecomment-1135520225) that has several DMS containers and local DNS config setup to test relaying. It may be helpful for you to experiment with?

When possible it's best to use `SASL` auth over ports 587 or 465 rather than sending to port 25. This will allow your mail to be forwarded/relayed when you authenticate successfully. [View our docs on current relay support](https://docker-mailserver.github.io/docker-mailserver/edge/config/advanced/mail-forwarding/relay-hosts/) for more information.

Otherwise I believe you may need to modify Postfix config based on their docs.
